### ISSUE TITLE
Runtime issue: Result for unknown rule. You may be running mismatch axe-core versions

### ISSUE THREAD

**Issue by somaalapati** [dequelabs/axe-core#4466]
<< Please describe how you encountered this issue >>

axe-core version: `4.9.1`
axe-extension version: `4.80.1`

```
Error: Result for unknown rule. You may be running mismatch axe-core versions
    at chrome-extension://hmbaabdipdenfbejpmnegelidhkbdabb/axe-versions/4.9.0/axe.js:2:478080
    at Array.map (<anonymous>)
    at sg.value (chrome-extension://hmbaabdipdenfbejpmnegelidhkbdabb/axe-versions/4.9.0/axe.js:2:478010)
    at chrome-extension://hmbaabdipdenfbejpmnegelidhkbdabb/axe-versions/4.9.0/axe.js:2:483000
    at chrome-extension://hmbaabdipdenfbejpmnegelidhkbdabb/axe-versions/4.9.0/axe.js:2:97092
    at chrome-extension://hmbaabdipdenfbejpmnegelidhkbdabb/axe-versions/4.9.0/axe.js:2:106685
    at chrome-extension://hmbaabdipdenfbejpmnegelidhkbdabb/axe-versions/4.9.0/axe.js:2:97092
    at chrome-extension://hmbaabdipdenfbejpmnegelidhkbdabb/axe-versions/4.9.0/axe.js:2:106617
    at chrome-extension://hmbaabdipdenfbejpmnegelidhkbdabb/axe-versions/4.9.0/axe.js:2:103254
    at post (chrome-extension://hmbaabdipdenfbejpmnegelidhkbdabb/content.bundle.js:2:19854)
```

**Comment by somaalapati** [comment]
This issue is raised when I scanned a webpage https://www.irctc.co.in/nget/train-search using axe extension. I didn't do anything special. We informed the same issue with the axe extension team thinking it may related to how extension handles axe-core.

**Comment by straker** [comment]
I'm able to load axe-core v4.9.1 directly on the page and perform a scan without issues.
